TransIT.BLL/Mappings/PartProfile.cs

```diff
@@ -0,0 +1,24 @@
+using AutoMapper;
+using TransIT.BLL.DTOs;
+using TransIT.DAL.Models.Entities;
+
+namespace TransIT.BLL.Mappings
+{
+    public class PartProfile : Profile
+    {
+        public PartProfile()
+        {
+            CreateMap<PartDTO, Part>()
+                .ForMember(e => e.UpdatedById, opt => opt.Ignore())
```

Comment:
You don't need to ignore `UpdatedById CreatedById`, because `PartDTO` doesn't have properties with these name, so they will be ignored automatically. Of course, if you want to ignore them you may.

You have to ignore properties, when they exist (same name) in both entities, which you are mapping, and you don`t want them to be mapped.